<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>460</width>
     <height>500</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1,1,2">
    <item>
     <widget class="QTextEdit" name="textEdit_serverContent">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>通讯协议</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_contactMethod">
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务器IP地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_serverIPAddress"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_serverPort">
        <property name="text">
         <string>8888</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>选择发送端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="myCombox" name="comboBox_ChoosePort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_startListen">
        <property name="text">
         <string>开始监听</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stopListen">
        <property name="text">
         <string>停止监听</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_lineout">
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit_serverSend"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_send">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myCombox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
